### ISSUE TITLE
Can't delete old/unused tokens

### ISSUE THREAD

**Issue by sleviim** [freeotp/freeotp-ios#326]
Trying to remove an unused token (was added wrongly so now it's duplicate).
There's no a clear "delete" option when pressing the token
swiping it right/left of the screen doesn't seem to have any effect

Using iphone 12
IOS version 16.6.1
FreeOTP 2.3.3
2013-2020 - Red Hat, Inc, et al.

**Comment by IvanVeloz** [comment]
You have to swipe _left_. Not right. Which is weird because it's usually to the right.

**Comment by delahondes** [comment]
Just in case someone come accross this issue, you can swipe the entry to the **right**. However the ergonomy of the gesture is a little touchy as a long press will trigger a change of relative position of the entry (and once in this mode you can't trigger the deletion). The gesture need to be light and quick and a large move is not required, even a quick narrow right move will do the trick and trigger the appearance of the question : "Remove token". 

Bottom line: Don't press long without moving, move your finger right quickly after you touch the screen on the entry you want to remove.
